### PR TITLE
[ MAMA Skillonomy ] SKILLONOMY-236: Apply the server language setting by default

### PR DESCRIPTION
**Pull request by OPersian**
**Description:** Set default lang to Russian; user prefs should apply.
See the ticket for more info.

**Youtrack:** https://youtrack.raccoongang.com/issue/SKILLONOMY-236

**Merge deadline:** 07 April

**Configuration instructions:** 

in "lms.env.json" this is to set "LANGUAGE_LANG" to "ru".

### CODE REVIEW

File: openedx/core/djangoapps/dark_lang/middleware.py [hunk 1]
```diff
@@ -118,13 +118,15 @@ def _clean_accept_headers(self, request):
...
+        since we need to rely solely on the server lang configs as per SKILLONOMY-236.
         """
         auth_user = request.user.is_authenticated()
-        preview_lang = None
-        if auth_user:
-            # Get the request user's dark lang preference
-            preview_lang = get_user_preference(request.user, DARK_LANGUAGE_KEY)
+        preview_lang = get_user_preference(
+            request.user, LANGUAGE_KEY
+        ) or settings.LANGUAGE_CODE if auth_user else settings.LANGUAGE_CODE
```
Comment (GlugovGrGlib) [inline openedx/core/djangoapps/dark_lang/middleware.py:129]: ```if auth_user else settings.LANGUAGE_CODE``` is unnecessary.

Comment (OPersian) [reply openedx/core/djangoapps/dark_lang/middleware.py:129]: if anonymous, we don't need to check user preferences. 

`get_user_preference` seems to be crafted for authenticated users

Comment (OPersian) [reply openedx/core/djangoapps/dark_lang/middleware.py:129]: I'm merging)

Comment (OPersian) [reply openedx/core/djangoapps/dark_lang/middleware.py:129]: Fixed with #1988

**Pull request merged**